spring-data-azure-documentdb/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.azure.java</groupId>
+    <artifactId>spring-data-azure-documentdb</artifactId>
+    <version>0.1.0</version>
+    <name>Azure Document DB Spring Data</name>
+    <description>Azure Document DB Spring Data Support</description>
+    <url>https://github.com/Microsoft/azure-spring-boot-starters</url>
+
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-spring-boot-dependencies</artifactId>
+        <version>0.1.1</version>
+        <relativePath>../azure-spring-boot-dependencies/pom.xml</relativePath> <!-- lookup parent from repository -->
+    </parent>
+
+    <scm>
+        <connection>scm:git:git://github.com/Microsoft/azure-spring-boot-starters.git</connection>
+        <developerConnection>scm:git:ssh://github.com:Microsoft/azure-spring-boot-starters.git</developerConnection>
+        <url>https://github.com/Microsoft/azure-spring-boot-starters/tree/master</url>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>yungez</id>
+            <name>Yunge Zhu</name>
+            <email>yungez@microsoft.com</email>
+        </developer>
+    </developers>
+
+    <properties>
+        <springdata.commons.version>1.13.4.RELEASE</springdata.commons.version>
```

Comment:
Declaring versions here should be used to override the version provided by spring-boot BOM